### PR TITLE
hi fofffa

### PR DESCRIPTION
**Pull request by sama9abudaf**

### CODE REVIEW

File: src/components/NavApp.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  const Search = styled("div")(({ theme }) => ({
+    background: "white",
+    width: "530px",
+    height: "40px",
+    borderRadius: "6px",
+    opacity: "1",
+  }));
+
+  const SearchIconButton = styled("div")(({ theme }) => ({
```
Comment (AbdHussein) [inline src/components/NavApp.jsx]: Let's move these styled-components to a separate folder. Create a folder named `styled` and put the following components inside in separated files for each component.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.MuiToolbar-root{
+    top: 0px;
+    left: 0px;
+   padding: 0px 70px;
+    height: 85px;
+    /* UI Properties */
+    background: #242424 0% 0% no-repeat padding-box;
+    opacity: 1;
+}
+.css-to4x1n-MuiBadge-badge{
```
Comment (AbdHussein) [inline src/index.css]: this class is useless because it's auto-generated by MUI at runtime, Find another way to apply this style.

**Review by AbdHussein — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 2 above)
Comment (sama9abudaf) [reply src/index.css]: O
kay doctor

File: src/components/NavApp.jsx (hunk 1 above)
Comment (sama9abudaf) [reply src/components/NavApp.jsx]: DONE SIR

**Review by AbdHussein — APPROVED** [review]

**Pull request merged**